core/src/main/scala/com/avsystem/commons/SharedExtensions.scala

```diff
@@ -188,6 +189,18 @@ object SharedExtensionsUtils extends SharedExtensions {
       }
   }
 
+  implicit class LazyOptOps[A](private val opt: () => Opt[A]) extends AnyVal {
```

Comment:
`implicit` no longer needed when you have a manual implicit conversion. It might even generate a conflict with unfortunate enough imports. Also make it `final` for additional performance optimizations by the JVM.